src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -469,19 +465,14 @@ function beginUpdateQueue(
 }
 exports.beginUpdateQueue = beginUpdateQueue;
 
-function commitCallbacks(finishedWork : Fiber, callbackList : UpdateQueue, context : mixed) {
-  const stopAfter = callbackList.last;
-  let update = callbackList.first;
-  while (update) {
-    const callback = update.callback;
-    if (typeof callback === 'function') {
```

Comment:
Also the type is stricter now. Previously we iterated through a list of `Update`s, whose `callback` type is `Function | null`, but now we're iterating through a list of functions.